tests/acceptance/features/apiSpaces/changeSpaces.feature

```diff
@@ -10,7 +10,7 @@ Feature: Change data of space
     Given user "Alice" has been created with default attributes and without skeleton files
     And the administrator has given "Alice" the role "Admin" using the settings api
 
-  Scenario: Alice changes a name of the space via the Graph api, she expects a 204 code and checks that the space name has changed
+  Scenario: Alice changes a name of the space via the Graph api, she expects a 200 code and checks that the space name has changed
```

Comment:
This is OK (needed fixing). In future, it will be better not to write so much detail in the scenario descriptions. Instead maybe just write "success code".

Or even just:
```suggestion
  Scenario: Alice can successfully change the name of a space via the Graph api
```